utils.py

```diff
@@ -22,25 +22,23 @@ def loadConfig(current_version):
 
 def saveConfig(config):
     with open("config.json", "w") as config_file:
-        json.dump(config, config_file)
+        json.dump(config, config_file, indent=2)
 
 
 def createConfig():
-    with open("config.json", "w") as config_file:
-        json.dump(
-            {
-                "theme": "Light",
-                "notify": False,
-                "ontop": False,
-                "transperency": 0.99,
-                "sound": True,
-                "default_minutes": 0,
-                "default_hours": 0,
-                "default_seconds": 5,
-                "sound_path": r".\assets\sounds\sound1.wav",
-            },
-            config_file,
-        )
+    saveConfig(
+        {
+            "theme": "Light",
+            "notify": False,
+            "ontop": False,
+            "transperency": 0.99,
+            "sound": True,
+            "default_minutes": 0,
+            "default_hours": 0,
+            "default_seconds": 5,
+            "sound_path": "assets/sounds/sound1.wav",

```

Comment:
`pathlib.Path` will convert this path to a native path, so we can use simple `/`-es

Reply:
Then on Windows? It uses backslashes, right?

Reply:
Yes. 👍